### PR TITLE
[loss refactor] [2] code style improvements

### PR DESCRIPTION
**Pull request by yueming-yuan**
## Summary

Follow-up to #753

### CODE REVIEW

File: miles/backends/training_utils/loss_hub/losses.py [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
-from .logits import get_log_probs_and_entropy, get_values
+
+class LossFunction(Protocol):
+    def __call__(
+        self,
+        args: Namespace,
+        batch: RolloutBatch,
+        logits: torch.Tensor,
+        sum_of_sample_mean: Callable[[torch.Tensor], torch.Tensor],
+    ) -> tuple[torch.Tensor, dict[str, torch.Tensor]]: ...
```
Comment (Shi-Dong) [inline miles/backends/training_utils/loss_hub/losses.py]: Might be better to replace `...` with a docstring for this protocol.

File: miles/backends/training_utils/loss_hub/logits.py [hunk 2]
```diff
@@ -3,11 +3,10 @@
 
 import torch
 
+from miles.backends.training_utils.cp_utils import _allgather_cp_redistribute, get_logits_and_tokens_offset_with_cp
```
Comment (Shi-Dong) [inline miles/backends/training_utils/loss_hub/logits.py]: I think a function is meant to be a module-private function if its name starts with an underscore. If we are to use `_allgather_cp_redistribute` in this module, we'd better rename it.

**Review by Shi-Dong — APPROVED** [review]
LGTM!  Two tiny nits.

**Review by Zhichenzzz — APPROVED** [review]
Lgtm!
